src/Components/Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL/AspireAzureEFPostgreSqlExtensions.cs

```diff
@@ -128,18 +128,11 @@ private static void ConfigureDbContextOptionsBuilder(AzureNpgsqlEntityFrameworkC
 
 #pragma warning restore EF1001 // Internal EF Core API usage.
 
-#if NET9_0_OR_GREATER
-        dbContextOptionsBuilder.UseNpgsql(connectionString, options =>
-        {
-            options.ConfigureDataSource(dataSourceBuilder => dataSourceBuilder.ConfigureEntraIdAuthentication(settings.Credential));
-        });
-#else
```

Comment:
Do the tests pass if you don't repass in the connection string (which seems wrong IMO)?

```csharp
#if NET9_0_OR_GREATER
        dbContextOptionsBuilder.UseNpgsql(options =>
        {
            options.ConfigureDataSource(dataSourceBuilder => dataSourceBuilder.ConfigureEntraIdAuthentication(settings.Credential));
        });
#else
```

Reply:
I confirm the issue still happens without the `connectionstring` argument. But what I found out is that I could repro the failing test when running two Enrich tests at the same time. they both use the same connection string, but if I have one of them use a different connection string (`Database=test2`) then I can't repro the problem.

Reply:
Filed an issue: https://github.com/npgsql/npgsql/issues/6085

Reply:
We have two options, refactor the tests on net9.0 to ignore this issue, or not use ConfigureDataSource in the net9.0 version of Enrich. I am fine with either.

Reply:
OK, this all makes sense - and is unfortunately by design... See https://github.com/npgsql/efcore.pg/issues/3513#issuecomment-2768406841 (am happy to discuss further).